includes/abstracts/abstract-wc-product.php

```diff
@@ -1048,7 +1048,7 @@ public function get_price_html( $price = '' ) {
 	 * @return string
 	 */
 	public function get_price_html_from_text() {
-		$from = '<span class="from">' . _x( 'From:', 'min_price', 'woocommerce' ) . ' </span>';
+		$from = '<span class="from">' . __( 'From:', 'woocommerce' ) . ' </span>';
```

Comment:
This is a BIG problem, since you can translate this difference in some languages if you know that this is `From:` is for price or time.